lib/install/config/webpack/shared.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   ),
 
   output: {
-    filename: '[name].js',
+    filename: '[name]-[hash].js',
```

Comment:
Yeah but not available with dev server and watcher actually, just production build